pandevice/userid.py

```diff
@@ -308,6 +321,11 @@ def get_registered_ip(self, ip=None, tags=None, prefix=None):
             cmd = ET.SubElement(cmd, "registered-address")
         else:
             cmd = ET.SubElement(cmd, "registered-ip")
+        sp = ET.SubElement(cmd, "start-point")
+        sp.text = str(start)
+        lim = ET.SubElement(cmd, "limit")
+        lim.text = str(limit)
+        ET.SubElement(cmd, "all")
```

Comment:
I haven't tested, but if you're going to unconditionally add in `<all></all>` to the XML sent, you need to make sure that this call still works when a user specifies a specific tag or ip.

Reply:
yes, you are probably right. I did test ansible requesting result for tags or specific IP-addresses and that seemed to give me the correct results.  But am not sure if the 'all' is actually needed. ( I guess it was used in the comments in issue #152.